Demo/Demo/DemoSettings/DemoSettings.swift

```diff
@@ -34,8 +34,13 @@ enum DemoSettings {
     }
 
     static var demoUIFramework: DemoUIFramework {
-        UserDefaults.standard.string(forKey: DemoUIFrameworkKey)
-            .flatMap { DemoUIFramework(rawValue: $0) } ?? .uikit
+        get {
+            UserDefaults.standard.string(forKey: DemoUIFrameworkKey)
+                .flatMap { DemoUIFramework(rawValue: $0) } ?? .uikit
+        }
+        set {
+            UserDefaults.standard.set(newValue.rawValue, forKey: DemoUIFrameworkKey)
+        }
```

Comment:
Why did we need to add this custom get and set?

Reply:
@scannillo we have on CardUITest that fails if its run on SwiftUI, but not on UIKit. So i made this getter in the demo app so that that test always runs in UIKit. Just a temporary solution until we get the demo in a good shape and start adding UItests